component/src/main/java/org/wso2/extension/siddhi/io/email/sink/EmailSink.java

```diff
@@ -410,6 +410,7 @@
     private Option optionTo;
     private Option optionCc;
     private Option optionBcc;
+    private Option optionContentType;
```

Comment:
Update the annotation doc with this new parameter.

Reply:
It's already been defined earlier in #L108